modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/resources/META-INF/resources/js/CKEditor4ReactClassicEditor.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {ClassicEditor} from 'frontend-editor-ckeditor-web';
+import React from 'react';
+
+const CKEditor4ReactClassicEditor = ({
```

Comment:
if this is a renaming of [ReactClassicEditor.tsx](https://github.com/liferay-frontend/liferay-portal/pull/4544/files#diff-da3880c6d35085e09b7b76fe4ba7b16c3fca8c5aace1dbad5554a3bb0691ba1d), perhaps we can rename the file so that git keeps the full history